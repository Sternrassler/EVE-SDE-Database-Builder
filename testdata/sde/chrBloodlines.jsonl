{"bloodlineID":1,"bloodlineName":"Deteis","raceID":1,"description":"Caldari bloodline focused on logic.","maleDescription":"Analytical and rational.","femaleDescription":"Analytical and rational.","shipTypeID":601,"corporationID":1000006,"perception":7,"willpower":5,"charisma":3,"intelligence":7,"memory":7,"iconID":null,"shortDescription":"Logical Caldari","shortMaleDescription":"Analytical","shortFemaleDescription":"Analytical"}
{"bloodlineID":2,"bloodlineName":"Brutor","raceID":2,"description":"Minmatar bloodline of warriors.","maleDescription":"Strong and resilient.","femaleDescription":"Strong and resilient.","shipTypeID":588,"corporationID":1000049,"perception":9,"willpower":7,"charisma":4,"intelligence":4,"memory":4,"iconID":null,"shortDescription":"Warrior Minmatar","shortMaleDescription":"Warriors","shortFemaleDescription":"Warriors"}
{"bloodlineID":4,"bloodlineName":"Amarr","raceID":4,"description":"Religious and traditional.","maleDescription":"Pious and noble.","femaleDescription":"Pious and noble.","shipTypeID":596,"corporationID":1000083,"perception":3,"willpower":10,"charisma":6,"intelligence":6,"memory":4,"iconID":null,"shortDescription":"Noble Amarr","shortMaleDescription":"Pious","shortFemaleDescription":"Pious"}
