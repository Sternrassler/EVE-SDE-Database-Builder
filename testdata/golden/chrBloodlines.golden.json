[
  {
    "bloodlineID": 1,
    "bloodlineName": "Deteis",
    "raceID": 1,
    "description": "Caldari bloodline focused on logic.",
    "corporationID": 1000006,
    "iconID": null,
    "shipTypeID": 601
  },
  {
    "bloodlineID": 2,
    "bloodlineName": "Brutor",
    "raceID": 2,
    "description": "Minmatar bloodline of warriors.",
    "corporationID": 1000049,
    "iconID": null,
    "shipTypeID": 588
  },
  {
    "bloodlineID": 4,
    "bloodlineName": "Amarr",
    "raceID": 4,
    "description": "Religious and traditional.",
    "corporationID": 1000083,
    "iconID": null,
    "shipTypeID": 596
  }
]